### PR TITLE
build(core): Upgraded all npm packages and rust crates

### PR DESCRIPTION
**Pull request by 0xterminator**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -23,34 +23,34 @@ version = "0.0.11"
 actix-cors = "0.7"
 actix-server = "2.5"
 actix-web = "4.9"
-anyhow = "1.0"
+anyhow = "1.0.91"
```
Comment (pedronauck) [inline Cargo.toml:26]: I guess we should follow the pattern of just using version until minor, by not fixing the patch we will have updates automatically in the lock.

```suggestion
anyhow = "1.0"
```

File: Cargo.toml [hunk 2]
```diff
@@ -23,34 +23,34 @@ version = "0.0.11"
...
 async-nats = "0.37"
 async-trait = "0.1"
-bytes = "1.7"
+bytes = "1.8.0"
```
Comment (pedronauck) [inline Cargo.toml:29]:
```suggestion
bytes = "1.8"
```

File: Cargo.toml [hunk 3]
```diff
@@ -23,34 +23,34 @@ version = "0.0.11"
...
+fuel-core-importer = { version = "0.40" }
+fuel-core-storage = { version = "0.40" }
+fuel-core-types = { version = "0.40", features = ["test-helpers", "serde"] }
+fuel-core-services = "0.40"
 futures-util = "0.3"
 pretty_assertions = "1.4"
 rand = "0.8"
-serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
+serde = { version = "1.0.213", features = ["derive"] }
```
Comment (pedronauck) [inline Cargo.toml:45]:
```suggestion
serde = { version = "1.0", features = ["derive"] }
```

File: Cargo.toml [hunk 4]
```diff
@@ -23,34 +23,34 @@ version = "0.0.11"
...
+serde_json = "1.0.132"
```
Comment (pedronauck) [inline Cargo.toml:46]:
```suggestion
serde_json = "1.0"
```

File: Cargo.toml [hunk 5]
```diff
@@ -23,34 +23,34 @@ version = "0.0.11"
...
 sha2 = "0.10"
 strum = "0.26"
 strum_macros = "0.26"
-tokio = { version = "1.40", features = ["full"] }
+tokio = { version = "1.41", features = ["full"] }
 tracing = "0.1"
-tracing-actix-web = "0.7"
-thiserror = "1.0"
+tracing-actix-web = "0.7.14"
```
Comment (pedronauck) [inline Cargo.toml:52]:
```suggestion
tracing-actix-web = "0.7"
```

File: Cargo.toml [hunk 6]
```diff
@@ -23,34 +23,34 @@ version = "0.0.11"
...
+thiserror = "1.0.65"
```
Comment (pedronauck) [inline Cargo.toml:53]:
```suggestion
thiserror = "1.0"
```

**Review by pedronauck — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (0xterminator) [reply Cargo.toml:26]: fixed

File: Cargo.toml (hunk 2 above)
Comment (0xterminator) [reply Cargo.toml:29]: fixed

File: Cargo.toml (hunk 3 above)
Comment (0xterminator) [reply Cargo.toml:45]: fixed

File: Cargo.toml (hunk 5 above)
Comment (0xterminator) [reply Cargo.toml:52]: fixed

File: Cargo.toml (hunk 6 above)
Comment (0xterminator) [reply Cargo.toml:53]: fixed

**Review by pedronauck — APPROVED** [review]

**Pull request merged**